cell_classification/segmentation_data_prep.py

```diff
@@ -100,7 +114,10 @@ def get_instance_masks(self, data_folder, cell_mask_key):
             np.array:
                 The instance mask
         """
-        return np.zeros([500, 500]), np.zeros([500, 500])
+        instance_mask = imread(os.path.join(data_folder, cell_mask_key + ".tiff"))
```

Comment:
It's generally the case with our data that the cell_mask_key is not the same as the image name for the segmentation mask. I would add another argument that's specific for the segmentation_mask_name